psydac/api/tests/build_domain.py

```diff
@@ -20,17 +20,12 @@ class TransposedPolarMapping(Mapping):
     _ldim        = 2
     _pdim        = 2
 
-def union(domains, name):
-    assert len(domains)>1
-    domain = domains[0]
-    for p in domains[1:]:
-        domain = domain.join(p, name=name)
-    return domain
-
-def set_interfaces(domain, interfaces):
+def create_domain(domains, interfaces, name):
+    connectivity = []
+    domains_interiors = [D.interior for D in domains]
     for I in interfaces:
-        domain = domain.join(domain, domain.name, bnd_minus=I[0], bnd_plus=I[1], direction=I[2])
-    return domain
+        connectivity.append(((domains_interiors.index(I[0].domain),I[0].axis, I[0].ext), (domains_interiors.index(I[1].domain), I[1].axis, I[1].ext), I[2]))
+    return Domain.join(domains, connectivity, name)
```

Comment:
Can you please rename `domains` as `patches`?

Reply:
done in [d66a68e](https://github.com/pyccel/psydac/pull/309/commits/d66a68ed7d030e41dc86823c805ed685b78d906e)